modules/social_features/social_landing_page/social_landing_page.install

```diff
@@ -780,7 +780,7 @@ function social_landing_page_update_11401(): string {
  * - "override landing_page revision log entry",
  * - "override landing_page promote to front landing_page option".
  */
-function social_landing_page_update_111101(): void {
```

Comment:
This change should land on 11.x as it it BC. Hook update number should not be changed.